cellarium/ml/train/training_plan.py

```diff
@@ -13,16 +13,22 @@
 
 class TrainingPlan(pl.LightningModule):
     """
-    Lightning module task to train cellarium-ml modules.
+    Lightning module task to train ``cellarium.ml`` modules.
 
     Args:
-        module: A cellarium.ml module to train.
-        optim_fn: A Pytorch optimizer class, e.g., :class:`~torch.optim.Adam`. If ``None``,
+        module:
+            A ``cellarium.ml`` module to train.
+        optim_fn:
+            A Pytorch optimizer class, e.g., :class:`~torch.optim.Adam`. If ``None``,
```

Comment:
Out of curiosity, what does Tilda do in this context?

Reply:
Tilde means to hide prefix in the class name and only display `Adam` in the docs.